window.go

```diff
@@ -54,6 +55,7 @@ const (
 	EventNameWindowEventDidGetRedirectRequest         = "window.event.did.get.redirect.request"
 	EventNameWindowEventWebContentsExecutedJavaScript = "window.event.web.contents.executed.javascript"
 	EventNameWindowEventWillNavigate                  = "window.event.will.navigate"
+	EventNameWindowEventUpdateCustomOptions           = "window.event.update.custom.options"
```

Comment:
Could you replace `update` with `updated` in the event name and the variable name?